torchtext/data/field.py

```diff
@@ -78,10 +78,15 @@ def __init__(
         self.pad_token = pad_token if self.sequential else None
 
     def preprocess(self, x):
-        """Load a single example using this field, tokenizing if necessary."""
+        """Load a single example using this field, tokenizing if necessary.
+        
+        If the input is a Python 2 `str`, it will be converted to Unicode
```

Comment:
maybe clearer to put `unicode` instead of Unicode, to make it explicit that we're referring to the [the python 2 unicode type?](https://docs.python.org/2/library/functions.html#unicode)